react/OrderBilling.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { useCssHandles } from 'vtex.css-handles'
+
+import { useOrder } from './components/OrderContext'
+
+const CSS_HANDLES = [
+  'billingAddressContainer',
+  'billingAddressTitle',
+  'billingName',
+  'billingPhoneNumber',
+  'billingStreet',
+  'billingCity',
+]
+
+interface Props {
+  displayTitle?: boolean
+}
+
+const OrderBilling: FC<Props> = ({ displayTitle = true }) => {
+  const { customData } = useOrder()
+  const handles = useCssHandles(CSS_HANDLES)
+
+  if (!customData || customData.customApps.length === 0) return null
```

Comment:
As the name says, CustomData varies from store to store. This component should be a custom component instead.

Reply:
Removed everything related billing from order-placed app and moved it to a custom app instead.